packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -80,6 +80,11 @@ export default class QuanticGeneratedAnswer extends LightningElement {
    * @default {false}
    */
   @api withToggle = false;
+  /**
+   * @internal
+   * The unique identifier of the answer configuration to use to generate the answer.
+   */
+  @api answerConfigurationId;
```

Comment:
why the `@internal` here? And where/how do we get this answerConfiguration from the config or is it manually passed to the component?

Also I would add a `@type ` decorator for this property as well

Reply:
It's internal because we don't wont this in our doc just yet, we want only when the feature will be officially GA